server/src/crew/positions.json

```
@@ -2,418 +2,387 @@
+  "Biochemistry Researcher",
+  "Biological Storage Specialist",
   "Biologist",
   "Biology Teacher",
-  "Biophysics Understudy",
-  "Botany Manager",
+  "Biotechnology Researcher",
+  "Biophysics Researcher",
+  "Botany Supervisor",
   "Botany Teacher",
-  "Botonist",
-  "Brig Officer",
+  "Botanist",
+  "Brig Attendant",
   "Bulkhead Maintenance Officer",
+  "Cartographer",
+  "CASM Generator Officer",
   "CASM Maintenance Officer",
-  "CRM 114 Control Officer",
+  "CASM Operator",
+  "CASM Operator Understudy",
+  "CRM 114 Chief",
   "CRM 114 Damage Control Officer",
+  "CRM 114 Maintenance",
   "CRM 114 Operator",
-  "CRM 114 Specialist",
-  "CRM 114 Understudy",
-  "CRM-114 Chief",
+  "CRM-114 Supervisor",
   "Cargo Loader",
   "Cargo Logger",
   "Cargo Supervisor",
-  "Cargo Transporter Chief",
-  "Cargo Transporter Officer",
+  "Cargo Transport Officer",
   "Cargobay Chief",
-  "Casm Generator Repairman",
-  "Casm Generator Understudy",
-  "Casm Operator",
   "Cellist",
   "Central Intelligence Officer",
   "Chaos Physics Lab Supervisor",
   "Chaos Physics Teacher",
-  "Chaos Theory Specialist",
   "Chaplain",
-  "Chaplain Aide",
   "Chef",
   "Chief Decryption Officer",
+  "Chief Legal Officer",
+  "Chief Medical Officer",
   "Chief Phaser Specialist",
   "Chief Probes Officer",
   "Chief Programmer",
   "Chief Railgun Specialist",
   "Chief Torpedo Specialist",
   "Clerk",
-  "Client / Server Manager",
   "Client Server Manager",
   "Cloaking Control Officer",
-  "Cloaking Device Understudy",
   "Cloaking Manager",
-  "Cloning Research Understudy",
+  "Cloning Researcher",
   "Cloning Teacher",
-  "Co2 Scrubber Control Officer",
-  "Communicaitons Crew Department Head",
+  "CO2 Scrubber Control Officer",
+  "Communications Department Head",
   "Communications Antenna Operator",
   "Communications Antenna Understudy",
   "Communications Array Understudy",
   "Communications Chief",
-  "Communications Crew Department Head",
-  "Communications Diagnostic Understudy",
+  "Communications Diagnostic Officer",
   "Communications Officer",
-  "Communications Specialist",
-  "Communications Sub Department Head",
+  "Communications Officer Understudy",
   "Communications Sub-department Head",
-  "Compressor Operator",
-  "Computer Control",
-  "Computer Core Chief",
+  "Computer Control Department Head",
+  "Computer Core Debugger",
   "Computer Core Maintenance",
-  "Computer Reinitilation Station",
-  "Contractor",
-  "Cook Understudy",
+  "Computer Reinitiation Station",
+  "Coroner",
+  "Cook"
+  "Cook Assistant",
   "Counselor",
-  "Crew Development Officer",
-  "CRM-114 Chief",
+  "Crew Dispatch Officer",
+  "Crime Scene Investigator",
   "Cryonic Specialist",
-  "Cryonic Teacher",
-  "Cryonic Understudy",
+  "Cryonic Specialists Understudy",
+  "Cryonic Researcher",
   "Cultural Studies Attendant",
+  "Custodial Crew Department Head",
   "Dance Instructor",
   "Data Storage",
-  "Decoder",
+  "Deccryptologist",
```

Cryptographer would be more accurate.
```suggestion
  "Cryptographer",
```